plugins/view-resources/src/components/ObjectPresenter.svelte

```diff
@@ -20,6 +20,7 @@
 
   export let objectId: Ref<Doc> | undefined = undefined
   export let _class: Ref<Class<Doc>> | undefined = undefined
+  export let attributeClass: Ref<Class<Doc>> | undefined = undefined
```

Comment:
don't see the difference between _class and attributeClass

should we just show nothing if objectId is undefined? there is very small chance that someone uses custom presenter for undefined value